e2e/test/iothub/AuthenticationWithTokenRefreshDisposalTests.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Azure.Devices.E2ETests
     public class AuthenticationWithTokenRefreshDisposalTests : E2EMsTestBase
     {
         public static readonly TimeSpan MaxWaitTime = TimeSpan.FromSeconds(10);
-        private readonly string _devicePrefix = $"E2E_{nameof(AuthenticationWithTokenRefreshDisposalTests)}_";
+        private readonly string _devicePrefix = $"{nameof(AuthenticationWithTokenRefreshDisposalTests)}_";
```

Comment:
The helper adds the E2E prefix. This is why we have so many test devices prefixed "E2E_E2E_".